src/services/io/podio/JEventProcessorPODIO.cc

```diff
@@ -135,6 +139,9 @@ JEventProcessorPODIO::JEventProcessorPODIO() {
             "CentralCKFSeededTrajectories",
             "CentralCKFSeededTracks",
             "CentralCKFSeededTrackParameters",
+            "CentralCKFSeededTrajectoriesFiltered",
+            "CentralCKFSeededTracksFiltered",
+            "CentralCKFSeededTrackParametersFiltered",
```

Comment:
Please do not add "*Filtered" collections for final results. We all believe that ambiguity solver is useful and should be the default. It's okay to omit the intermediate collections after track finder before solver from the default list of collections, so their names would need to be only mentioned in tracking.cc.